compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeSceneFocusManager.skiko.kt

```diff
@@ -66,4 +69,27 @@ class ComposeSceneFocusManager internal constructor(
      * Release focus from [ComposeScene].
      */
     fun releaseFocus() = measureAndLayoutThen { focusOwner().releaseFocus() }
+
+    /**
+     * If [position] is outside the bounds of the currently focused node, clear focus.
+     */
+    fun clearFocusIfOutsideOfActiveFocusTargetNode(position: Offset) = measureAndLayoutThen {
```

Comment:
The `ComposeSceneFocusManager` is a facade for manipulating `ComposeScene` focus by external clients. This method isn't needed for external clients and only needed for `ComposeScene` itself.

Let's move this method into `BaseComposeScene` and make it private.

You will need accessing `focusOwner` - you can make a protected abstract method.